lib/rules/media-feature-name-value-whitelist/README.md

```diff
@@ -0,0 +1,65 @@
+# media-feature-name-value-whitelist
+
+Specify a whitelist of allowed media feature and value pairs.
+
+```css
+@media screen and (min-width: 768px) {}
+/**                    ↑        ↑
+ *         These features and these values */
+```
+
+**Caveat:** Media feature names within range or boolean context are ignored.
```

Comment:
Please write as the following to be consistent with most of the other rules:

"This rules ignores media feature within range or boolean context."

I'll create a separate issue about updating the other rules that still use the old "**Caveat:**" convention (e.g. `media-feature-name-blacklist`).

Reply:
Yeah, I reused some things here and there to the rule as consistent as possible. That the caveat was deprecated, I did not know :)

Reply:
> Yeah, I reused some things here and there to the rule as consistent as possible.

Yep, I saw. Good job and thanks :)

> That the caveat was deprecated, I did not know :)

We're still ironing out all the kinks in the docs, which is why your kind of feedback is so useful.

We just merged [a PR](https://github.com/stylelint/stylelint/pull/3360) to make the other rule READMEs consistent.